clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -2881,7 +2881,7 @@ private void verifySessionPartitions() {
                                 field.setAccessible(true);
                                 LinkedHashMap<?, ?> sessionPartitions =
                                         (LinkedHashMap<?, ?>) field.get(handler);
-                                for (Map.Entry<?, ?> entry : sessionPartitions.entrySet()) {
+                                for (@SuppressWarnings("UnusedLocalVariable") Map.Entry<?, ?> ignored : sessionPartitions.entrySet()) {
```

Comment:
I tried

```
sessionPartitions.forEach((key, value) -> Thread.yield());
```

and the checkstyle did not complain. It seems that the unused check does not care about lambda arguments.

Reply:
Done, also in some other place